pkgs/tools/networking/linkchecker/default.nix

```diff
@@ -1,22 +1,49 @@
-{ stdenv, lib, fetchurl, python2Packages, gettext }:
-
+{ stdenv, lib, fetchFromGitHub, fetchpatch, python2, gettext }:
+let
+  # pin requests version until next release.
+  # see: https://github.com/linkcheck/linkchecker/issues/76
+  python2Packages = (python2.override {
+    packageOverrides = self: super: {   
+      requests = super.requests.overridePythonAttrs(oldAttrs: rec {
+        version = "2.14.2";
+        src = oldAttrs.src.override {
+          inherit version;
+          sha256 = "0lyi82a0ijs1m7k9w1mqwbmq1qjsac35fazx7xqyh8ws76xanx52";
+        };
+      });
+    };
+  }).pkgs;
+in
 python2Packages.buildPythonApplication rec {
-  name = "LinkChecker-${version}";
-  version = "9.3";
+  pname = "LinkChecker-${version}";
```

Comment:
`pname` should not include the version. `name` will be set to `"${pname}-${version}"` automatically